### ISSUE TITLE
Menu options in 'Help' menu do nothing

### ISSUE THREAD

**Issue by paulschwoerer** [keepassxreboot/keepassxc#10147]
## Overview

Hello,

thank you very much for your work on KeepassXC!

I found a small issue, I'd like to let you know about. Some Options in the 'Help' menu do nothing for me. Specifically: 'Getting Started', 'User Guide' and 'Keyboard Shortcuts'.

Unfortunately I could not find KeepassXC log files. If you let me know where to find them in a Flatpak install, I'd be happy to share them.

## Steps to Reproduce
1. Start KepassXC
2. Click on one of the aforementioned buttons

## Expected Behavior
A new window should probably be opened

## Actual Behavior
Nothing happens

## Context

KeePassXC - Version 2.7.6
Revision: dd21def
Distribution: Flatpak

Operating System: Linux
Desktop Env: Gnome
Windowing System: X11

**Comment by droidmonkey** [comment]
Yes unfortunately that appears to be an issue with the way Qt launches local files within a flatpak or snap sandbox. We will have to switch to using xdg-open directly.
